composer.json

```diff
@@ -49,7 +49,8 @@
         "symfony/dom-crawler": "^2.8|^3.0|^4.0",
         "symfony/yaml": "^2.8|^3.0|^4.0",
         "friendsofphp/php-cs-fixer": "^1.1|^2.8@dev",
-        "lcobucci/jwt": "~3.2"
+        "lcobucci/jwt": "~3.2",
+        "phpunit/phpunit": ">=4.0"
```

Comment:
why is that needed? `phpunit-bridge` embeds it

Reply:
Nope https://github.com/symfony/phpunit-bridge/blob/master/composer.json

Reply:
Got it. You mean `simple-phpunit`. You are right, removed then.